### ISSUE TITLE
Discussion: Priority of algorithm usage across different Twitter features

### ISSUE THREAD

**Issue by jonasbeyer** [twitter/the-algorithm#1449]
**_tl;dr: less algorithms for home feed generation, more algorithms for content & creator discovery across features like Lists, Topics, Search and Spaces_**

I think Twitter, a **text**-focused service, shouldn't deliver a "For you" feed that contains any "embedding space" or a large amount of "social graph" content, apart from likes & retweets, like other popular social networks.

Instead, it's more natural to provide fine-grained ways to discover follow-worthy accounts and noteable tweets through better use of
- a user's topic subscriptions details
- Twitter's home-built "SimClusters" system

as well as
- highlighting platform-wide popular tweets and accounts 
- improvements in search. 

It's very unlikely that an algorithm can outperform a human being in picking whose tweets best fit in one's personal feed and content moderation happens most efficently by following the right and unfollowing the wrong people.

**Comment by jonasbeyer** [comment]
Currently, the Twitter "Explore" page only shows trends and tweets from topics that a user is subscribed to. 

I think it's the best place to present everything Twitter has to offer: most viewed and liked tweets in the last 24 hours, tweets and accounts related to the following list or the user's view history, most frequently subscribed Topics and most listened to Spaces, largest businesses and their executives, top athletes, SimClusters communities and so on. Of course, these categories should also be sortable and searchable.

That's how a user can easily disable recommended tweets/out of networt content on the home timeline without losing the ability to discover new things.

**Comment by jonasbeyer** [comment]
Twitter's "For you" feed is an absolute disaster for people who follow 2000+ people. :D 
I was hoping that solving this problem had a higher priority and I still think the approach outlined above is the best.

![Screenshot 2023-05-19 134217](https://github.com/twitter/the-algorithm/assets/7658962/964fedfd-07a8-4b66-9260-29c956716e8e)

**Comment by twister21** [comment]
### Harmonization (ToS) and technical integration of YouTube, Twitter, Reddit, Quora, StackExchange, Medium/Substack, Discord and LinkedIn under the X services umbrella

Product design and governance principles should be harmonized across all participating services.

Requirements:

- accounts should be interoperable to allow users to follow the same accounts and topics across all supported services, as well as to facilitate further helpful data sharing at the user's discretion
- guaranteed access to public content via apps and APIs
- maximally automated indexing/labelling of user-generated content
- no avoidable pop-ups and banners
- improved feed and customization options (see algorithm strategy proposed above)

All involved sites would benefit greatly from an open and well-integrated information system.
